sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -1579,6 +1580,27 @@ def test_lull_logging(self):
         '.*There has been a processing lull of over.*',
         'Unable to find a lull logged for this job.')
 
+@attr('ValidatesRunner')
+class FnApiBasedStateBackedCoderTest(unittest.TestCase):
+  def create_pipeline(self):
+    return beam.Pipeline(
+        runner=fn_api_runner.FnApiRunner(use_state_iterables=True))
+
+  def test_state_backed_coder(self):
+    class MyDoFn(beam.DoFn):
+      def process(self, gbk_result):
+        value_list = gbk_result[1]
+        return (gbk_result[0], sum(value_list))
+
+    with self.create_pipeline() as p:
+      # The number of integers could be a knob to test against
+      # different runners' default settings on page size.
+      main = (p | 'main' >> beam.Create([('a', 1) for _ in range(0, 20000)])
+              | 'GBK' >> beam.GroupByKey()
+              | 'Sum' >> beam.ParDo(MyDoFn()))
```

Comment:
This could just be beam.MapTuple(lambda key, values: sum(values))

Reply:
Actually, a better test would be to ensure no more than N (for some value of N < number of elements) instances of the value type are alive at any given moment.

Reply:
Maybe let us keep both. 

Would you please explain how to control some values 'alive' while others in the same iterable not?

Reply:
So what you'd want to do is create some class with custom pickling (implement `__reduce__`). This picking would be artificially large, e.g. include a dummy `'a' * 1000` value, as many runners trigger on serialized size not element count.

In the constructor, you would increment a class-level variable to indicate how many are alive. In the destructor (`__del__`), you would decrement it. An error would be thrown if there are too many.

Reply:
Thanks. Updated based on the suggestions above.